r4e-img-upgrade-iso.sh

```diff
@@ -0,0 +1,59 @@
+# script for creating an update ISO
```

Comment:
the script is missing the ISO part ;-)

Reply:
in the upgrade flow we don't need an iso
and if this is a new edge creation the first part would build a most updated image (checked it)